Analysis/config/2017/sources.cfg

```diff
@@ -2,7 +2,7 @@
 int_lumi: 41529
 period: Run2017
 mode: HH
-qcd_shape: OS_AntiIsolated
+qcd_shape_str: OS_AntiIsolated
```

Comment:
don't change how the param is called in the config. keep `qcd_shape`. Same for 2016&2018 configs.